lib/pymedphys/tests/mosaiq/test_mock_patient_table.py

```diff
@@ -1,68 +1,175 @@
+from pymedphys._imports import numpy as np
 from pymedphys._imports import pandas as pd
 from pymedphys._imports import pymssql, pytest, sqlalchemy
 
-import pymedphys.mosaiq
+from pymedphys._mosaiq.helpers import get_patient_fields, get_patient_name
+from pymedphys.mosaiq import connect, execute
 
+msq_server = "."
+test_db_name = "MosaiqTest77008"
 
-@pytest.mark.mosaiqdb
-def test_mock_patient_table():
-    msq_server = "."
-    test_db_name = "MosaiqTest77008"
+sa_user = "sa"
+sa_password = "sqlServerPassw0rd"
+
+# set up a SQLAlchemy engine to be used for populating tables
+connection_str = f"mssql+pymssql://{sa_user}:{sa_password}@{msq_server}/{test_db_name}"
+engine = sqlalchemy.create_engine(connection_str, echo=False)
```

Comment:
All of these calls up here happen on import, so even if the `--mosaiqdb` flag is not used this code here will still run. Can the engine be made as a fixture instead?